assets/stylesheets/common/common.scss

```diff
@@ -1522,9 +1522,9 @@ body.composer-open .topic-chat-float-container {
     border: 2px solid var(--primary-low);
     border-left: transparent;
     .chat-channel-row-controls {
-      bottom: -235px;
+      bottom: unset;
```

Comment:
You probably don't need to unset the `bottom` value, it doesn't look like it is set elsewhere.

Reply:
ah.. I believe you are correct!